README.md

```diff
@@ -431,3 +431,5 @@ To get started:
 
 1.  `yarn start` here to watch for local changes to `react-showtime`,
 2.  `yarn start` in `example/` to watch for changes to the demo site and run a dev server.
+
+This project uses the [Git Flow](https://nvie.com/posts/a-successful-git-branching-model/) branching workflow. Usage of the [`gitflow-avh`](https://github.com/petervanderdoes/gitflow-avh) CLI tool is recommended for releases and hotfixes.
```

Comment:
Added this in here as a placeholder prior to a `CONTRIBUTING` file being created. I also wanted to mention our preferred version of the `gitflow` CLI. There was some confusion as to where the commands in the release issue template were coming from.